From 600aabb1a2235396db5fce4240ac0751258fcf7f Mon Sep 17 00:00:00 2001
From: Guillem Jover <guillem@debian.org>
Date: Sun, 18 Sep 2022 18:05:57 +0200
Subject: [PATCH 1/6] copyright-format: Place Source field after Format field

These both contain URLs, are the most important references, and nicely
align, so it's nice to get them as early as possible in the file.
---
 copyright-format-1.0.xml | 8 ++++----
 1 file changed, 4 insertions(+), 4 deletions(-)

diff --git a/copyright-format-1.0.xml b/copyright-format-1.0.xml
index d29c490..3d5c672 100644
--- a/copyright-format-1.0.xml
+++ b/copyright-format-1.0.xml
@@ -271,9 +271,9 @@
       <section id="example-header-paragraph">
         <title>Example header paragraph</title>
 <programlisting>Format: https://www.debian.org/doc/packaging-manuals/copyright-format/1.0/
+Source: https://www.example.com/software/project
 Upstream-Name: SOFTware
-Upstream-Contact: John Doe &lt;john.doe@example.com&gt;
-Source: https://www.example.com/software/project</programlisting>
+Upstream-Contact: John Doe &lt;john.doe@example.com&gt;</programlisting>
       </section>
     </section>
 
@@ -1265,8 +1265,8 @@ also delete it here.</programlisting>
         package):
 <programlisting><![CDATA[
 Format: https://www.debian.org/doc/packaging-manuals/copyright-format/1.0/
-Upstream-Name: X Solitaire
 Source: ftp://ftp.example.com/pub/games
+Upstream-Name: X Solitaire
 
 Files: *
 Copyright: 1998 John Doe <jdoe@example.com>
@@ -1306,9 +1306,9 @@ License: GPL-2+
         <literal>planet-venus</literal> package):
 <programlisting><![CDATA[
 Format: https://www.debian.org/doc/packaging-manuals/copyright-format/1.0/
+Source: https://www.example.com/code/venus
 Upstream-Name: Planet Venus
 Upstream-Contact: John Doe <jdoe@example.com>
-Source: https://www.example.com/code/venus
 
 Files: *
 Copyright: 2008, John Doe <jdoe@example.com>
-- 
2.37.2

